modules/virt-about-reviewing-resource-consumption.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assembly:
+//
+// * virt/logging_events_monitoring/virt-about-reviewing-resource-consumption-by-pods.adoc
+//
+
+[id="virt-about-reviewing-resource-consumption-by-pods_{context}"]
+
+= About reviewing resource consumption by pods
+
+Line chart graphs in the {VirtProductName} dashboard present data for virtual machines that are consuming the most resources within each pod for a selected time range.
+
+The following table shows monitored resources and describes the metrics associated with each resource for top consumers in each pod.
+
+[cols="1,1"]
+|===
+|Monitored resources| Description
+
+|Memory swap traffic
+|Virtual machines consuming the most memory pressure when swapping memory.
+|vCPU wait
+|Virtual machines experiencing the maximum wait time (in seconds) for their vCPUs.
+|CPU usage by pod
+|Pod with virtual machines that are using the maximum CPU usage.
+|Network traffic
+|Virtual machines that are saturaing the network by receiving the most amount of network traffic (in bytes).
+|Storage traffic
+|Virtual machines with the highest amount (in bytes) of storage-related traffic.
+|Storage IOPS
+|Virtual machines with the highest amount of I/O operations per second over a time period.
+|Memory usage
+|Virtual machines that are consuming the most memory.
```

Comment:
@lmandavi Virt-launcher pods, that each one is running a Virtual Machine, and are using the most memory.